src/components/atoms/inputTime/inputTime.scss

```
@@ -1,11 +1,22 @@
 input[type=time] {
     padding: $small-space;
     border: 1px solid $gray-light;
-    border-radius: 2px;
+    border-radius: $regular-radius;
+    
+    &:hover {
+      background-color: $primary-utility-color;
+      border: 1px solid $primary-dark-color;
+      border-radius: $regular-radius;
+    }
+
+    &:focus {
+      border: 2px solid $gray-lighter;
+      border-radius: $regular-radius;
+    }
 }
 
 
 input[type=time]::-webkit-calendar-picker-indicator {
   background-color: $secondary-utility-color;
   background: url("../../../../public/icons/chevron.svg") no-repeat center;
-}
+}
```

```suggestion
}

```
manque un retour a la ligne